src/components/access-type-display/access-type-display.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { KeyValue, NoValue } from '@folio/stripes/components';
+
+import { getAccessTypeId } from '../utilities';
+
+const propTypes = {
+  accessStatusTypes: PropTypes.object.isRequired,
+  model: PropTypes.object.isRequired,
+};
+
+function AccessTypeDisplay({ model, accessStatusTypes }) {
+  const accessTypesRecords = accessStatusTypes.resolver.state.accessTypes.records;
+  const accessTypesAreEmpty = !Object.keys(accessTypesRecords).length;
+
+  if (accessTypesAreEmpty) {
+    return null;
+  }
```

Comment:
I would better make this check in the parent component. No need to call `AccessTypeDisplay` if there is no access status types